src/rest/SearchEndpoint.ts

```diff
@@ -339,6 +342,11 @@ export class SearchEndpoint implements ISearchEndpoint {
         this.logger.error('Please update your REST Search API');
       }
 
+      // Transform the duration compared to what the search API returns
+      // We want to have the "duration" to be the time as seen by the browser
+      results.searchAPIDuration = results.duration;
+      results.duration = TimeSpan.fromDates(begun, new Date()).getMilliseconds();
```

Comment:
Maybe a better prop name for `duration` is `browserDuration` or `e2eDuration` allowing to remove the comments.

Reply:
I like simply "duration" because that's : 

1) What appears in the documentation as the "most important" duration (because this is what the query truly took for the end user). Most important duration == short/concise name.

2) Browser duration could imply "duration that it took to render", for example.

3) I don't like e2eDuration because the abbreviation makes it harder to know what it is. Especially since it's a publicly documented property (so it is official).